prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-adjudicator-documents/enrollee-adjudicator-documents.component.ts

```diff
@@ -45,6 +48,14 @@ export class EnrolleeAdjudicatorDocumentsComponent implements OnInit {
       );
   }
 
+  public onDeleteDocumentById(documentId: number) {
+    this.busy = this.enrolmentResource.deleteEnrolleeAdjudicationDocument(this.enrolleeId, documentId)
+      .subscribe((document: EnrolleeAdjudicationDocument) => {
+        this.getDocuments();
+        this.toastService.openSuccessToast('Document has been deleted');
```

Comment:
If we put general toasts into the resource files we reduce the need to replicate the toast if the resource request is used in multiple places then the subscribe can be a one line arrow function.